dask/dataframe/io/parquet/utils.py

```diff
@@ -462,14 +462,12 @@ def _scrub(i, p):
                     break
             basepath = basepath[:j]
         l = len(basepath)
-
     else:
         basepath = _join_path(root).split("/")
         l = len(basepath)
         assert all(
             p[:l] == basepath for p in path_parts_list
         ), "All paths must begin with the given root"
-    l = len(basepath)
```

Comment:
It looks like `l` is used a few lines down in `"/".join(path_parts[l:])`

Reply:
yea, it's set in both the `if` and `else` branches above though, so this line is redundant